python/MufluxDigiReco.py

```diff
@@ -239,19 +238,10 @@ def digitizeMuonTagger(self):
 			#station -= 1 #offset - 6 volumes defined in the geo file but 1 not sensitive
 		assert station in range(1,6) #limiting the range of rpcs
 		
-		#z coordinates of every station 
-		if station == 1: zcoord = 874.25
-		elif station == 2: zcoord = 959.25
-		elif station == 3: zcoord = 1004.25
-		elif station == 4: zcoord = 1049.25
-		elif station == 5: zcoord = 1094.25
-
                 #calculate strip
                 # x gives vertical direction
                 direction = 1
                 strip = StripX(xcoord) 
-		if strip < 0 and strip > 185 :
-			break 
```

Comment:
I would move these checks into StripX/Y instead of removing them completely. Maybe as assertions.

Reply:
I'm not sure how to limit the values without throwing an error - that's why this was in the main event loop before

Reply:
I don't see anything wrong with throwing an error (e.g. through an insertion), as silent failure could be worse. Just make sure it gives an understandable error (e.g. `assert should_be_true == True, "somethings wrong"`)